doc/use-status-backend-server.md

```diff
@@ -0,0 +1,53 @@
+## Solution to use Status Backend Server
```

Comment:
@qfrank, I'm sorry for such a late after-merge comment. I think this doc would benefit from quick introduction - defining what is `Status Backend Server` and what purpose it serves.